appc.js

```diff
@@ -9,11 +9,13 @@ module.exports = {
     platforms: {
         ios: {
             vendor: {
+
+                // Q: How would we make sure this is unique accross modules?
                 'MyFramework': {
                      // these can be an array or string
-                     source: ['src'],
-                     header: 'src',
-                     resource: 'src'
+                     source: ['vendor'],
+                     header: 'vendor',
+                     resource: 'vendor'
```

Comment:
If we do that change, we would also need to change it in the `appc.js` located in normal projects, because I guess the CLI will merge these files together.